bokun.yaml

```diff
@@ -5952,14 +6271,62 @@ definitions:
         description: 'Only used if hasOpeningHours == true. The default opening hours by weekday. These may be overridden by seasonal opening hours.'
       seasonalOpeningHours:
         type: array
-        description: 'Optional seasonal opening hours. If date falls within the season then this will override the default opening hours.'
+        description: 'Deprecated. Optional seasonal opening hours. If date falls within the season then this will override the default opening hours.'
         items:
           $ref: '#/definitions/SeasonalOpeningHours'
+      displaySettings:
+        type: object
+        properties:
+          showPickupPlaces:
+            type: boolean
+            description: 'Specifies if pick up places are shown.'
+          showRouteMap:
+            type: boolean
+          selectRateBasedOnStartTime:
+            type: boolean
+          customFields:
+            type: array
+            items:
+              type: object
+            description: 'Deprecated.'
+      hasBoxes:
+        type: boolean
+        description: 'Not working.'
+      requestDeadline:
+        type: integer
+        format: int64
+        description: 'The number of miliseconds allocated to confirm or reject booking by activity supplier.'
+      actualId:
+        type: integer
+        format: int64
+        description: 'Specififies ID of the original owner of the product.'
+      bookingCutoff:
+        type: integer
+        format: int32
+        description: 'The number of minutes before the activity when we stop accepting bookings.'
+      nextDefaultPriceAsText:
+        type: string
+        description: 'String of the price (e.g. "USD 15.90") for 1 person in the default price category, assuming the future departure date nearest in time to the current time. Amount in default locale currency.'
```

Comment:
String of the price -> Human readable price

Reply:
fixed